samples/math.k

```diff
@@ -0,0 +1,3 @@
+import math
```

Comment:
Just add the example into unit test cases.

Reply:
I think this use case can be added here. The use case under '/sample' represents the minimum capability that the product should have. 'math.k' can be used to judge whether the built product can find the runtime environment.